src/main/java/jenkins/branch/OrganizationFolder.java

```diff
@@ -282,13 +300,24 @@ public boolean isSingleOrigin() {
     }
 
     /**
+     * The {@link BranchBuildStrategy}s to apply.
+     *
+     * @return The {@link BranchBuildStrategy}s to apply.
+     * @since 2.0.12
+     */
+    public DescribableList<BranchBuildStrategy, BranchBuildStrategyDescriptor> getBuildStrategies() {
```

Comment:
🐜  @CheckForNull due to deserializing old format.

Reply:
nope. Top level project type, so constructor is always called and then unmarshal on top